EncoreTickets.SDK/Pricing/Models/JsonDate.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace EncoreTickets.SDK.Pricing.Models
+{
+    public class JsonDate
```

Comment:
Why does the name of this object have "json"? I don’t understand why this might depend on JSON, and this name is also not from the documentation for the pricing API

Reply:
This is for endpoints that return date not as a string with a pre-defined date format, but as a JSON object

Reply:
It looks like any object that can be returned even if the data is returned, for example, not in json, but in xml. So let's not bind the name with the format in which the data was received, but simply name this object by date-related and contained fields. For example, YearMonthDate or something similar, I think it would be better